tests/idea-space.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect } from '@playwright/test';
+import { api } from './utils';
+
+let ideaId = 0;
+
+test.describe('/api/idea-cards', () => {
+
+    test("should add idea", async ({ request }) => {
+        const newIdea = await api(request).post("/api/idea-cards", {
+            ideaName: "Testing",
+            tagline: "Yay!",
+            description: "I am testing",
+            targetAudience: "Yes",
+            features: "Yes",
+            experience: "Yes",
+            extraInfo: "Yes",
+            involveLevel: "minimum",
+            status: "workshopping",
+            author: 1,
+            ideaOwner: 1,
+        });
+        expect(newIdea.attributes.ideaName).toBe("Testing");
+        ideaId = newIdea.id;
```

Comment:
The ideaId should also be 1 if it's the first idea?

Reply:
Could be, but I don't want to rely on it. Technically, I could read userId too, to avoid using id directly.